indexd/utils.py

```diff
@@ -47,7 +47,7 @@ def setup_database(
     """Setup the user and database"""
 
     if not no_drop:
-        try_drop_test_data(database)
+        try_drop_test_data(database=database, root_user=root_user, host=host)
```

Comment:
I think I'm missing something: does this change anything? The `setup_database` gets called [here|https://github.dev/NCI-GDC/indexd/blob/5932bddbc27048855e55735e1522f08cadd50222/tests/conftest.py#L39] without any parameters, are we planning to provide `root_user` and `host` in the future?

Reply:
The utils of indexd is not only used by tests of indexd, it is currently also used by most repos that needs indexd server for tests.